deep_speech_2/data_utils/data.py

```diff
@@ -306,7 +319,25 @@ def _padding_batch(self, batch, padding_to=-1, flatten=False):
             padded_audio[:, :audio.shape[1]] = audio
             if flatten:
                 padded_audio = padded_audio.flatten()
-            new_batch.append((padded_audio, text))
+
+            padded_instance = [padded_audio, text]
+            padded_conv0_h = (padded_audio.shape[0] - 1) // 2 + 1
+            padded_conv0_w = (padded_audio.shape[1] - 1) // 3 + 1
```

Comment:
Please add some comments for the hard-coded parameters here

Reply:
Done.